src/neo/SmartContract/InteropService.Encode.cs

```diff
@@ -0,0 +1,52 @@
+using Neo.Cryptography;
+using Neo.IO;
+using Neo.VM;
+using Neo.Wallets;
+using System;
+using System.Linq;
+
+namespace Neo.SmartContract
+{
+    partial class InteropService
+    {
+        public static class Encode
```

Comment:
It's under Encode , but the method it's `Crypto`

Reply:
> It's under Encode , but the method it's `Crypto`

![image](https://user-images.githubusercontent.com/43407364/74817227-894c3100-5337-11ea-9ba1-ec1f469b1303.png)
Some parallel classes like InteropService.Crypto.cs are also defined in this manner

Reply:
> It's under Encode , but the method it's `Crypto`

Have changed accordingly